apps/admin_app/lib/admin_app_web/views/product_view.ex

```diff
@@ -29,7 +31,12 @@ defmodule AdminAppWeb.ProductView do
   end
 
   def has_themes(parent_product) do
-    length(parent_product.taxon.variation_themes) > 0
+    length(parent_product.taxon.variation_themes) > 0 && !check_if_child(parent_product)
+  end
+
+  defp check_if_child(parent_product) do
+    child_products = from(c in Variation, select: c.child_product_id) |> Repo.all()
```

Comment:
Count the number of records to validate the condition. 
Check logic in `is_parent_product(product_id)` function `AdminAppWeb.ProductView` module.